doc/source/whatsnew/v0.18.1.txt

```diff
@@ -43,8 +43,8 @@ Enhancements
 
 API changes
 ~~~~~~~~~~~
-
-
+- ``searchsorted`` for ``Index`` and ``TimedeltaIndex`` now accept a ``sorter`` argument to maintain compatibility with numpy's ``searchsorted`` function (:issue:`12238`)
+- The parameter for specifying values to be inserted into a ``pandas`` object when calling its ``searchsorted`` method is now uniformly denoted as ``value``
```

Comment:
Done.  I put `[ci skip]` in my commit because this is only a doc change.  So it should be good to merge if there is nothing else.